host-configs/lassen-blueos_3_ppc64le_ib_p9-clang@8.0.1_cuda.cmake

```diff
@@ -97,17 +97,17 @@ set(BLT_CMAKE_IMPLICIT_LINK_DIRECTORIES_EXCLUDE "/usr/tce/packages/gcc/gcc-4.9.3
 
 # Root directory for generated TPLs
 
-set(TPL_ROOT "/usr/WS1/axom/libs/blueos_3_ppc64le_ib_p9/2022_05_11_12_00_28/clang-8.0.1" CACHE PATH "")
+set(TPL_ROOT "/usr/WS1/axom/libs/blueos_3_ppc64le_ib_p9/2022_06_23_21_51_07/clang-8.0.1" CACHE PATH "")
 
 set(CONDUIT_DIR "${TPL_ROOT}/conduit-0.8.3" CACHE PATH "")
 
 set(C2C_DIR "${TPL_ROOT}/c2c-1.3.0" CACHE PATH "")
 
-set(MFEM_DIR "${TPL_ROOT}/mfem-4.2.0" CACHE PATH "")
+set(MFEM_DIR "${TPL_ROOT}/mfem-4.4.0" CACHE PATH "")
 
 set(HDF5_DIR "${TPL_ROOT}/hdf5-1.8.22" CACHE PATH "")
 
-set(LUA_DIR "${TPL_ROOT}/lua-5.3.5" CACHE PATH "")
+set(LUA_DIR "/usr/WS1/axom/libs/blueos_3_ppc64le_ib_p9/2022_06_23_21_51_07/clang-9.0.0/lua-5.3.5" CACHE PATH "")
```

Comment:
So, in the BlueOS Clang 8.0.1 build, Lua requires Clang 9?

Reply:
I can't explain this one -- I think spack found the two to be equivalent and used a cached copy (?)
I double-checked that all the builds/tests are passing on the RZ host-configs (where it made the same choice).

I spoke to @white238 about it yesterday, and he was seeing similar behavior on other projects. 
I was thinking about leaving it as is for this PR (since it is passing all the tests) 
and pinging the spack team to see if they had thoughts about it.
Would that work?

Reply:
It is fine with me to leave LUA_DIR as-is for this PR.  I am very interested in what the Spack team says.